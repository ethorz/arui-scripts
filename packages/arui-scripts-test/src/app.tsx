import React from 'react';
import image from './clock.svg';
import { isSmaller } from './utils';

import styles from './app.module.css';
import './style.css';

export class App extends React.Component {
    state = {
        clickCount: 0
    };

    render() {
        return (
            <div className={ styles.root }>
                <h2>
                    Hello, arui-scripts!
                </h2>

                Check hot-loader: <br />
                Button is clicked { this.state.clickCount } times
                Clicked more than 10 times? <br />
                { isSmaller(this.state.clickCount, 10) }

                <button
                    className='btn'
                    onClick={ () => this.setState({ clickCount: this.state.clickCount + 1 }) }
                >
                    Up!
                    <img src={ image } className='icon' />
                </button>
            </div>
        )
    }
}
