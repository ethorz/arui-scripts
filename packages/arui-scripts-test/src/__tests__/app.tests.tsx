import { shallow } from 'enzyme';
import React from 'react';
import { App } from 'Src/app';

describe('app', () => {
  it('should be match to snapshot', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
