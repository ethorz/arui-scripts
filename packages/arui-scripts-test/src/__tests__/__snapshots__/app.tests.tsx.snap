// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should be match to snapshot 1`] = `ShallowWrapper {}`;
